tests/test_web.py

```
@@ -15,3 +21,38 @@ def test_load_homepage(client, settings, mocker):
 
 	response = client.get("/")
 	assert response.status_code == 200
+
+
+def test_htmlhead_hreflang():
+	rf = RequestFactory()
+	request = rf.get("/cards/613/default/")
+	head = HTMLHead(request)
+
+	head.set_hreflang(lambda lang: "/cards/613/x-%s/" % lang)
+
+	hreflang_tags = [
+		t for t in head.get_tags() if t.tag_name == "link" and "hreflang" in t.attrs
+	]
+
+	def get_lang_tag(lang):
+		for tag in hreflang_tags:
+			if tag.attrs["hreflang"] == lang:
+				return tag
+		return None
```

return None is implied

---

This should raise an exception (or `assert False` I guess) instead.